packages/vm/tests/tester.js

```diff
@@ -16,8 +16,14 @@ function runTests() {
 
   const FORK_CONFIG = (argv.fork || config.DEFAULT_FORK_CONFIG)
   const FORK_CONFIG_TEST_SUITE = config.getRequiredForkConfigAlias(FORK_CONFIG)
-  // Examples: Istanbul -> istanbul, MuirGlacier -> muirGlacier
-  const FORK_CONFIG_VM = FORK_CONFIG.charAt(0).toLowerCase() + FORK_CONFIG.substring(1)
+
+  let FORK_CONFIG_VM
+  if (FORK_CONFIG.toLowerCase() == "dao") {
+    FORK_CONFIG_VM = "homestead"
```

Comment:
This is correct? If so, can you add a comment here?

Reply:
On a second look: this is just for the blockchain tests, right? Can we then move this over there and leave the simple logic here untouched? I think this is better understandable then in this context.

Reply:
I should add a DAO fork to hardforks support of VM, don't know why I did this actually 😅